npm/mount-utils/README.md

```diff
@@ -8,6 +8,140 @@ It is used in:
 
 - [`@cypress/react`](https://github.com/cypress-io/cypress/tree/develop/npm/react)
 - [`@cypress/vue`](https://github.com/cypress-io/cypress/tree/develop/npm/vue)
+- [`@cypress/svelte`](https://github.com/cypress-io/cypress/tree/develop/npm/svelte)
+- [`@cypress/angular`](https://github.com/cypress-io/cypress/tree/develop/npm/angular)
+
+## What is a Mount Adapter?
+
+All Component Tests require a component to be **mounted**. This is generally done with a custom command, `cy.mount` by default.
+
+
+### Requirements
+
+All the functionality used to create the first party Mount adapters is available to support third parties adapters. At minimum, a Mount Adapter must:
+
+- Receive a component as the first argument. This could be class, function etc - depends on the framework.
+- Return a Cypress Chainable (for example using `cy.wrap`) that resolves whatever is idiomatic for your framework
```

Comment:
I would add:
```
- Teardown the last mounted component between tests
```

and update the example to show the teardown.

Reply:
Added your point - good idea.

I have [this line](https://github.com/cypress-io/cypress/pull/23976/files#diff-e5219acad52acdefe7159f0202d3fac79431644def4dcc106a3f70db4b8be0deR54)

```js
Cypress.on("test:before:run", () => {
    // Do some cleanup from previous test
  });
```

There actually isn't any teardown to be done here. The one thing I'd *like* to do is unregister the Web Component - after a solid google, though, this is currently unsupported! :(

Any other ideas/recommendations we could add for a cleanup example?